platform/windows/display_server_windows.cpp

```diff
@@ -735,7 +735,7 @@ Error DisplayServerWindows::_file_dialog_with_options_show(const String &p_title
 		GetWindowRect(fd->hwnd_owner, &crect);
 		fd->wrect = Rect2i(crect.left, crect.top, crect.right - crect.left, crect.bottom - crect.top);
 	} else {
-		fd->hwnd_owner = nullptr;
+		fd->hwnd_owner = (HWND)INVALID_HANDLE_VALUE;
```

Comment:
In this case, we probably want to add a check for the invalid value around `SendMessage(fd->hwnd_owner` calls, since it's not intended to receive either `NULL` or `INVALID_HANDLE_VALUE`.